packages/babel-helper-define-polyfill-provider/src/imports-cache.ts

```diff
@@ -48,7 +45,10 @@ export default class ImportsCache {
       source: t.StringLiteral,
       // eslint-disable-next-line no-undef
       name: t.Identifier,
-    ) => { node: t.Node, name: string },
+    ) => {
+      node: t.Node;
+      name: string;
+    },
```

Comment:
```suggestion
    ) => { node: t.Node; name: string },
```

I know that this is an unnecessary nitpick and that it's done by flowts, but unnecessary newlines in small objects annoy me :sweat_smile:

Reply:
I think it's done by the Prettier portion of `flowts`, but I'll try changing it to this and see if Prettier complains

Reply:
Or not, guess it was a `recast` thing or something? guess Prettier has no opinion there, so I was able to revert it without issues. Same for the newlines added `ObjectMap` in `corejs3/src/built-in-definitions.ts`